package.json

```diff
@@ -152,8 +152,7 @@
         "standard-version": "9.5.0",
         "ts-mockito": "2.6.1",
         "ts-node": "10.9.2",
```

Comment:
Should we drop?

```
  "@taiga-ui/browserslist-config": "0.328.0",
```

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/12411/commits/5f4b12a89018bbb3c85473e1d78666ddd65564e3